### PR TITLE
Use round avatar for E.Cash room avatar

### PR DESCRIPTION
**Pull request by yuwenmemon**
@TomatoToaster please review

### Details
Just changes the Avatar from being square to round

### Fixed Issues
$ https://github.com/Expensify/Expensify/issues/170939

### Tests/QA
- Join / View a default chat room from the side bar, make sure the chair icon is round. 

### Tested On

- [x] Web
- [x] Mobile Web
- [x] Desktop
- [x] iOS
- [x] Android

#### Web
Screen Shot 2021-07-19 at 10 11 10 PM

#### Mobile Web
Screen Shot 2021-07-19 at 10 19 28 PM

#### Desktop
Screen Shot 2021-07-19 at 10 16 39 PM

#### iOS
Screen Shot 2021-07-19 at 11 38 22 PM

#### Android
Screen Shot 2021-07-19 at 9 59 04 PM

### CODE REVIEW

**Comment by yuwenmemon** [comment]
I have read the CLA Document and I hereby sign the CLA

File: src/styles/styles.js [hunk 1]
```diff
@@ -1191,7 +1191,6 @@ const styles = {
     avatarNormal: {
         height: variables.componentSizeNormal,
         width: variables.componentSizeNormal,
-        backgroundColor: themeColors.icon,
```
Comment (yuwenmemon) [inline src/styles/styles.js:1194]: Without removing this, the background would appear square:
Screen Shot 2021-07-19 at 11 27 23 PM

Comment (TomatoToaster) [reply src/styles/styles.js:1194]: Coool and it doesn't seem to affect the other icons. LGTM

**Review by TomatoToaster — APPROVED** [review]

**Comment by TomatoToaster** [comment]
This passed qa so I'm checking this off

**Pull request merged**